src/components/language-select/LanguageSelect.scss

```diff
@@ -1,5 +1,20 @@
 .custom-language-select {
   font-weight: bold !important;
+  position: relative;
+}
+
+.custom-language-select:focus-within {
+  box-shadow: none;
+}
+
+.custom-language-select:focus-within::after {
+  border: 2px solid var(--color-white);
+  bottom: -2px;
```

Comment:
I would add "display: block" just to avoid lineheight issues and just for clarity.

Reply:
Ah yes. I'll fix it. Thanks. 👍